content/about.md

```diff
@@ -274,6 +274,89 @@ The title says it all.
 <!-- /marekt -->
 
 
+### Bardaci
+-----------
+
+The name **Bardaci** is just garbled Slovak word 'babraci', which we can
+translate as 'numb hand'. Although that's not always true, we have used it
+since Mr. Gazi called so the first original **Bardaci** (those two below).
+So it's really hard to describe one without the other and vice versa, they have
+come neck by neck and both are similarly crazy.
+
+But give it a try:
+
+<!-- adamh -->
+<div class="photo" markdown="1">
+
+![Herda](/img/avatars/adam_h.png)
+
+</div>
+
+<div class="about" markdown="1">
+#### Adam Herda {#herda}
+*Bardak 1*
+
+He prepares (and sometimes destroys) hardware, but his place is primarily at
+the table with a soldering iron in one hand. We can say he is kind of Jakub's
+successor.
+
+In addition to the title *Bardak 1* he is also *master of merkur*, especially
+merkur robotic arm. And so you see how necessary is he in our team - you never
+know when will you need next servant-arm.
+
+</div>
+<div class='clear'></div>
+<!-- /adamh -->
+
+
+<!-- adamm -->
+<div class="photo" markdown="1">
+
+![Mikula](/img/avatars/adam_m.png)
+
+</div>
+
+<div class="about" markdown="1">
+#### Adam Mikula {#mikula}
+*Bardak 2*
+
+Every time he walks into our small lab, he brings a lot of fun and talks with him.
```

Comment:
`Every time he walks into our small lab he brings a lot of fun and talks to Adam Herda`